products/BellHybrid/apps/Application.cpp

```diff
@@ -24,16 +24,17 @@ namespace app
                              std::string parent,
                              StatusIndicators statusIndicators,
                              StartInBackground startInBackground,
-                             uint32_t stackDepth,
+                             std::uint32_t stackDepth,
                              sys::ServicePriority priority)
-        : ApplicationCommon(name, parent, statusIndicators, startInBackground, stackDepth, priority)
+        : ApplicationCommon(
+              std::move(name), std::move(parent), statusIndicators, startInBackground, stackDepth, priority)
     {
         getPopupFilter().addAppDependentFilter([&](const gui::PopupRequestParams &popupParams) {
-            bool val = ((isCurrentWindow(gui::popup::resolveWindowName(gui::popup::ID::Reboot))) ||
```

Comment:
I am not in favor of changing all types to `auto`. IMO, e.g. the `bool` type is more readable in this place.

Reply:
Changed back to `bool`